src/components/card/DataCard.tsx

```diff
@@ -15,10 +15,11 @@ type Props = {
     | 'warning'
     | 'danger'
     | 'dark';
+  isAChild?: boolean;
 };
 
-const DataCard = ({ children, title, theme }: Props) => (
-  <Card theme={`${theme}`}>
+const DataCard = ({ children, title, theme, isAChild = false }: Props) => (
+  <Card isAChild={isAChild} theme={`${theme}`}>
```

Comment:
isAChild mało mówi, już lepiej daj coś typu: withShadow czy coś. Na razie to będzie lepsze

Reply:
👍